### PR TITLE
use config file or argv to enable/disable streaming the following apis devices,ports,links,alarms

### PR DESCRIPTION
**Pull request by ananalaghbar**

### CODE REVIEW

File: ufm-stream-to-fluentd/README.md [hunk 1]
```diff
@@ -56,7 +56,11 @@ Run
 | --ufm_username <ufm_username> | [ufm-server-config.username](ufm-stream-to-fluentd.sample.cfg#L8) | True | Username of UFM user
 | --ufm_password <ufm_password> | [ufm-server-config.password](ufm-stream-to-fluentd.sample.cfg#L9) | True | Password of UFM user
 | --logs_file_name <logs_file_name> | [logs-config.logs_file_name](ufm-stream-to-fluentd.sample.cfg#L24) | False | Log file name, if not provided a default stream wil lbe used
-| --logs_level <logs_level> | [logs-config.logs_level](ufm-stream-to-fluentd.sample.cfg#L27) | False | Default is 'info' 
+| --logs_level <logs_level> | [logs-config.logs_level](ufm-stream-to-fluentd.sample.cfg#L27) | False | Default is 'info'
+| --streaming_systems <logs_level> | [streaming-config.systems](ufm-stream-to-fluentd.sample.cfg#L12) | False | Default is 'True'
```
Comment (haithamwj) [inline ufm-stream-to-fluentd/README.md:60]: what is logs_level?

File: ufm-stream-to-fluentd/README.md [hunk 2]
```diff
@@ -56,7 +56,11 @@ Run
...
+| --streaming_ports <logs_level> | [streaming-config.ports](ufm-stream-to-fluentd.sample.cfg#L13) | False | Default is 'True'
+| --streaming_links <logs_level> | [streaming-config.links](ufm-stream-to-fluentd.sample.cfg#L14) | False | Default is 'True'
+| --streaming_alarms <logs_level> | [streaming-config.alarms](ufm-stream-to-fluentd.sample.cfg#L15) | False | Default is 'True'
```
Comment (haithamwj) [inline ufm-stream-to-fluentd/README.md:63]: what is logs_level here?

Comment (haithamwj) [reply ufm-stream-to-fluentd/README.md:63]: > by mistake, fixed in
> [f2d2b7a#diff-54a3d44ecaf2ddb9ddf6463ddcea804c709e075ff64396e2675311e8ab80f206R62](https://github.com/Mellanox-lab/ufm_sdk/commit/f2d2b7a980e2d6ecd9b16cc66d54cb49940ce565#diff-54a3d44ecaf2ddb9ddf6463ddcea804c709e075ff64396e2675311e8ab80f206R62)

each commit should be fixed at the same pull request we made the comment on

Comment (ananalaghbar) [reply ufm-stream-to-fluentd/README.md:63]: sure

**Review by haithamwj — APPROVED** [review]

**Pull request merged**